EndlessClient/Controllers/InventoryController.cs

```diff
@@ -348,29 +354,41 @@ public void TradeItem(EIFRecord itemData, InventoryItem inventoryItem)
                     EOResourceID.DIALOG_TRANSFER_OFFER);
             }
         }
-
         private void DoItemDrop(EIFRecord itemData, InventoryItem inventoryItem, Action<int> dropAction,
-                                ItemTransferDialog.TransferType transferType = ItemTransferDialog.TransferType.DropItems,
-                                EOResourceID message = EOResourceID.DIALOG_TRANSFER_DROP)
+                        ItemTransferDialog.TransferType transferType = ItemTransferDialog.TransferType.DropItems,
+                        EOResourceID message = EOResourceID.DIALOG_TRANSFER_DROP)
         {
+            bool playDefaultSound = true;
+
             if (inventoryItem.Amount > 1)
             {
+                if (inventoryItem.ItemID == 1 && inventoryItem.Amount > 10000 && transferType == ItemTransferDialog.TransferType.DropItems)
```

Comment:
If you want to break the condition out of the if statement, do something like

`var isLargeGoldItemDrop = inventoryItem.ItemID == 1 && inventoryItem.Amount > 10000 && transferType == ItemTransferDialog.TransferType.DropItems;`

I think this should also move closer to where it's used since I'm asking you to remove the `playDefaultSound` parameter from the item transfer dialog factory too.

Reply:
![image](https://github.com/ethanmoffat/EndlessClient/assets/70440468/7a0f94fb-0cf2-4065-81dc-811c8082b70a)